Dockerfile

```
@@ -1,13 +1,39 @@
+FROM python:3.10.1-alpine
```

This matches the version of Python on `cisagov/saver:1.2.5`.

```suggestion
FROM python:3.10.8-alpine
```

---

Good catch!